src/o365/graph/commands.ts

```diff
@@ -23,6 +23,7 @@ export default {
   O365GROUP_USER_LIST: `${prefix} o365group user list`,
   O365GROUP_USER_REMOVE: `${prefix} o365group user remove`,
   SCHEMAEXTENSION_ADD: `${prefix} schemaextension add`,
+  REPORTS_TEAMS_DEVICE_USAGE_USER_DETAIL: `${prefix} reports teamsdeviceusageuserdetail`,
```

Comment:
Let's change the command name to `report` to align with other commands where we use the noun in singular. Sorry I didn't catch it earlier

Reply:
Fixed and ready for review.